### PR TITLE
Add gum recipe

### PR DESCRIPTION
**Pull request by cartoonist**
This PR adds a recipe for gum library.

### CODE REVIEW

**Comment by cartoonist** [comment]
@BiocondaBot please add label

File: recipes/gum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - {{ compiler('cxx') }}
+    - zlib
```
Comment (bgruening) [inline recipes/gum/meta.yaml:37]: zlib should go to the host section

**Comment by cartoonist** [comment]
Since it is not yet merged, I updated the recipe to use the newer version of the library without changing the build number.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**